### ISSUE TITLE
Question: I see 'eventemitter3' in the dependancies, and you do not using it anywhere

### ISSUE THREAD

**Issue by jamil-tlbwork** [santiq/bulletproof-nodejs#41]

**Comment by jamil-tlbwork** [comment]
Question: I see `eventemitter3` in the dependancies, and you do not using it anywhere

**Comment by jaschaio** [comment]
I wondered the exact same thing. Instead it looks like he is using the native node event dispatcher in https://github.com/santiq/bulletproof-nodejs/blob/master/src/decorators/eventDispatcher.ts

**Comment by santiq** [comment]
Yep, I'm sorry guys, I ended up using the package `event-dispatch` instead of `eventemitter3` because it has the decorators for Typescript.

But since `event-dispatch` is now deprecated (see #3 ), I want to change it to `eventemitter3`
